jwql/instrument_monitors/miri_monitors/data_trending/15min_to_db.py

```diff
@@ -0,0 +1,141 @@
+import statistics
+import os
+import jwql.instrument_monitors.miri_monitors.data_trending.utils.mnemonics as mn
+import jwql.instrument_monitors.miri_monitors.data_trending.utils.sql_interface as sql
+import jwql.instrument_monitors.miri_monitors.data_trending.utils.csv_to_AstropyTable as apt
+from jwql.utils.utils import get_config, filename_parser
+
+from jwql.instrument_monitors.miri_monitors.data_trending.utils.process_data import once_a_day_routine
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+#point to the directory where your files are located!
+directory = '/home/daniel/STScI/trainigData/set_15_min/'

```

Comment:
This should point to the central store location (via config.json)

Reply:
@bourque do you have a sufficient path for me?

Reply:
@DanielKuebi Yes, but actually we shouldn't even need it here.  As @johannes-sahlmann suggests, it should be stored and read in from the `config.json` file, i.e.:

```python
from jwql.utils.utils import get_config

directory = get_config()['miri_data_trending']   # or name it whatever you think is best
```

and in the `config.json` file:

```json
{
    "other_stuff" : "foo",
    "miri_data_trending" : "/path/to/files"
}
```

In that way, you can store it as `/home/daniel/STScI/trainigData/set_15_min/` on your end, and we can store it as our central store location on our end.